src/pages/en/index.astro

```diff
@@ -0,0 +1,175 @@
+                </label>
+                <br />
+              </>
+            ))}
+          </fieldset>
+        </form>
+      ))
+    }
+    <div class="flex justify-center">
+      <button
+        id="output-button"
+        class="py-4 px-8 text-lg font-bold rounded-lg border-none shadow bg-amber-400 hover:cursor-pointer hover:shadow-lg transition-all"
+      >
+        Generate Steam Review!</button
+      >
+    </div>
+
+    <div class="flex justify-center mt-4">
+      <button
+        id="copy-output"
+        class="py-2 px-4 text-lg rounded-lg border-none shadow bg-sky-100 hover:cursor-pointer hover:shadow-lg transition-all"
+      >
+        Copy output</button
+      >
+    </div>
+
+    <pre id="output"></pre>
+  </div>
+
+  <script>
+    var data = {};
+    import config_json from "../../review_templates/template_en.json";
+
+    function showOutput() {
+      copyButton?.style.setProperty("display", "block");
+      let output = document.getElementById("output");
+      if (output) {
+        output.innerHTML = "";
+        config_json.categories.forEach((category) => {
+          output.innerHTML += `\n---{ ${category.title} }---\n`;
+          category.options.forEach((item) => {
+            if (
+              category.type === "check" &&
+              data[category.title].includes(item)
+            ) {
+              output.innerHTML += `☑ ${item}\n`;
+            } else if (data[category.title] === item) {
+              output.innerHTML += `☑ ${item}\n`;
+            } else {
+              output.innerHTML += `☐ ${item}\n`;
+            }
+          });
+        });
+
+        output.innerHTML += `\n`;
+        output.innerHTML += `---{ Author }---\n`;
+        output.innerHTML += `☑ https://vojtastruhar.github.io/steam-review-template \n`;
+      }
+    }
+
+    let outputButton = document.getElementById("output-button");
+    if (outputButton) {
+      outputButton.addEventListener("click", showOutput);
+    }
+    let copyButton = document.getElementById("copy-output");
+    if (copyButton) {
+      copyButton.addEventListener("click", () => {
+        navigator.clipboard.writeText(
+          document.getElementById("output").innerText
+        );
+      });
+    }
+    copyButton?.style.setProperty("display", "none");
+
+    [...document.getElementsByTagName("input")].forEach((input) => {
+      if (input.type == "checkbox") {
+        data[input.name] = [];
+      }
+      input.addEventListener("change", (event) => {
+        let target = event.target;
+        if (target) {
+          if (target.type == "checkbox") {
+            if (target.checked) {
+              data[target.name].push(target.value);
+            } else {
+              data[target.name] = data[target.name].filter(
+                (item) => item != target.value
+              );
+            }
+          } else {
+            data[target.name] = target.value;
+          }
+        }
+      });
+    });
+  </script>
```

Comment:
Now that we have more than 1 language, this can definitely be refactored. I'd say it's going to be a function in a separate `.js` file with a language argument? Can Astro do that?